### ISSUE TITLE
UITableView scroll position reset after interactive pop

### ISSUE THREAD

**Issue by dennishylau** [fastred/SloppySwiper#28]
FIrst thing first, thank you for your work!

In the gif below, notice how the "Police Officer" row appears after popping the details page, even though I have already scrolled pass it beforehand. The UITableView always goes back to the same scroll position, no matter where I am before a segue.

<img src="https://user-images.githubusercontent.com/30386036/49016695-d76c9600-f1c1-11e8-97e5-415bc88e4211.gif" height="480" width="270">

**Comment by romaHerman** [comment]
@dennishylau , do you have reload data call in your table view inside viewWillAppear or viewDidAppear methods ? 

if yes you can use native API isMovingToParentViewController == false => reloadData 
or you can use your own custom flag to distinguish between this

**Comment by dennishylau** [comment]
@romaHerman No, reloadData was not called. Also the weird thing is, even if I store the contentOffset of the table view and set it back in viewWillAppear, it still won't work.

**Comment by ccrama** [comment]
I am seeing this behavior as well
